coraza_test.go

```diff
@@ -26,8 +26,8 @@ func TestPlugin(t *testing.T) {
 		t.Fatal(err)
 	}
 	res, _ := tester.AssertGetResponse(baseURL+"/test", 200, "test123")
-	if len(res.Header.Get("x-unique-id")) == 0 {
```

Comment:
With the caddy dependency v2.6.2, setting up `newTester("test.init.config", t)` was leading to  [`tc.t.Skipf("skipping tests as failed integration prerequisites. %s", err)`](https://github.com/caddyserver/caddy/blob/v2.6.2/caddytest/caddytest.go#L119). And tests were skipped

Reply:
I tried to figure out why was this change needed. Could you point it out?

Reply:
The test is executed by running Caddy with [test.init.config](https://github.com/corazawaf/coraza-caddy/blob/main/test.init.config) file. This file has the following line: `header * x-request-id "{http.transaction_id}"` that manipulates the response header adding one called `x-request-id`.
I think that the purpose of this line was asserting that this header is present (it fails if it is missing `t.Error("X-Unique-Id header is not set")`), but the name of the header seems wrong. Therefore, this change is renaming `X-Unique-Id` to `X-Request-Id`.

This test never failed because of (what I guess was a) [caddy bug](https://github.com/caddyserver/caddy/pull/5079), and tests were skipped rather than failing. Updating Caddy raised the problem